css/src/components/gradient-card.scss

```diff
@@ -0,0 +1,11 @@
+.gradient-card {
+	border: $border-width-md solid $border !important;
+	border-radius: $border-radius-lg !important;
```

Comment:
We usually create variables with `!default` flag that holds the value to use within CSS rules so if needed it's easier to update the value in the project using Atlas. For example, here it would be this:

```scss
$gradient-card-border-radius: $border-radius-lg !default;

.gradient-card {
   ...
   border-radius: $gradient-card-border-radius;
   ...
}
```

And later if needed it can be overwritten like this and the new value would be applied to that project:

```scss
$gradient-card-border-radius: 125px;
```

Please create vars for all rules that potentially can be adjusted (border, border-radius). Also, check out any other component file for reference.

Reply:
Thanks for the guidance! Updated.